src/ui/opengl/castletiledmap_map.inc

```diff
@@ -1034,6 +1034,18 @@ begin
     end;
   finally FreeAndNil(I) end;
 
+  { TODO : Handle tiles which are not explicitly found in the tsx-file.
+    Hint : Tiles which have no special setting (probability equals 1.0,
+           no terrains set, no type set) are not explicitly written down
+           in the tsx file. This means they are not loaded by the iteration
+           above and consequently are not found in the Tiles list. }
+  if not (TileCount = Tiles.Count) then
+  begin
+    WritelnWarning(URL + ': Could not load all tiles of tileset');
```

Comment:
Show URL using `URIDisplay(URL)`.

Also make it a single warning, not 2 warnings. This is desired in case some code captures warnings, counts them, sends them somewhere (e.g. to a remote server) etc. -- so a long warning should still be 1 warning, not 2 warnings :)

And in general use format (`%d` instead of `IntToStr` concatenation) -- it looks a bit better for longer strings.

So like

```
WritelnWarning('%s: Could not load all tiles of a tileset. Expected tiles: %d, loaded tiles: %d', [
  URIDisplay(URL),
  TileCount,
  Tiles.Count
]);
```

Reply:
The following files can be used in the Tiled editor. Anyway, if you look at the tsx-file you will find, that only one tile is explicitly found (with id = 2). For this specific tile I changed the probability to illustrate this. All the Tiled examples used in CGE have modified tiles in the tileset, hence they can be loaded. 

![test512](https://user-images.githubusercontent.com/16453692/124845957-1aa55d00-df98-11eb-829c-b323d1ee6b90.png)
[Test512-missing-tiles.tsx.txt](https://github.com/castle-engine/castle-engine/files/6780748/Test512-missing-tiles.tsx.txt)
[map512-missing-tiles.tmx.txt](https://github.com/castle-engine/castle-engine/files/6780749/map512-missing-tiles.tmx.txt)